### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by percurnicus**

### CODE REVIEW

File: tests/test_planetary_test_data.py [hunk 1]
```diff
@@ -7,9 +7,123 @@
...
+
+def test_final_path_2():
+    # Test when tests/mission_data/ exists but data.json Does Not Exist (DNE)
+    os.chdir('test_2')
+    assert os.path.exists(os.path.join('tests', 'mission_data'))
+    assert planetary_test_data.setup_json_file() == data_path_1
+    os.remove(data_path_1)
+    assert not(os.path.exists(data_path_1))
+    os.chdir('..')
+    assert os.path.basename(os.getcwd()) == 'tests'
```
Comment (percurnicus) [inline tests/test_planetary_test_data.py:56]: The seemingly excessive asserts is my attempt to test any additional code I needed to run the test

File: planetary_test_data/planetary_test_data.py [hunk 2]
```diff
@@ -1 +1,66 @@
 # -*- coding: utf-8 -*-
+import os
+import json
+import urllib
+import planetary_test_data
+import shutil
+
+"""
```
Comment (godber) [inline planetary_test_data/planetary_test_data.py]: Please include both of these comments as "docstrings" https://www.python.org/dev/peps/pep-0257/ in their functions.

File: tests/test_planetary_test_data.py [hunk 3]
```diff
@@ -7,9 +7,123 @@
...
+def test_default_json():
+    default_path = os.path.abspath(planetary_test_data.__file__)
+    default_json = os.path.join(os.path.dirname(default_path), 'data.json')
+    test_path = os.path.join('planetary_test_data', 'planetary_test_data',
+                             'data.json')
+    assert re.search(test_path, default_json)
+
+
+def test_final_path_1():
+    # Test when tests/mission_data/data.json exists
```
Comment (godber) [inline tests/test_planetary_test_data.py]: Convert these comments to docstrings.

**Comment by godber** [comment]
Why are there `.IMG` files in the test directories?  Are they required?  I can't think of any reason why we'd want them included in the repo, the entire point of this package to to keep `.IMG` files out of our repositories.

**Comment by percurnicus** [comment]
I thought `get_mission_data` was supposed to download the image files to mission_data directory? I included an image file in one of the test directories to see if `test_planetary_data`  1) will not download the file if it already exists, and 2) download the file when it doesn't.

**Comment by godber** [comment]
Yeah, I guess you're right.  I would have chosen smaller files, but this will work.

**Comment by percurnicus** [comment]
I was planning on editing the supported images list and replace larger images with smaller images. I can choose smaller images for these as well after I do that.

**Comment by godber** [comment]
Don't bother replacing them for space saving reasons.  If you delete the existing `.IMG` files they will still remain in the repository, and hence take up disk space.

**Pull request merged**